api/edge_api/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
@matthewelwell  should I remove this file?

Reply:
I think you probably can, yes.